src/main/java/dinkplugin/notifiers/ExternalPluginNotifier.java

```diff
@@ -87,7 +89,7 @@ private void handleNotify(ExternalNotificationRequest input) {
         var player = Utils.getPlayerName(client);
         var template = Template.builder()
             .template(input.getText())
-            .replacements(input.getReplacements())
+            .replacements(Objects.requireNonNullElse(input.getReplacements(), new HashMap<>(2)))
```

Comment:
if we're microoptimization heads, `requireNonNullElseGet` could be used, but not necessary (or a simple ternary)